packages/commonwealth/client/scripts/views/components/component_kit/cw_component_showcase.tsx

```diff
@@ -2214,6 +2211,20 @@ export const ComponentShowcase = () => {
           </div>
         </CWDrawer>
       </div>
+      <div className="auth-buttons">
```

Comment:
how about using new components page? :) we should not add code to deprecated page